package.json

```diff
@@ -40,8 +40,8 @@
     "release:patch": "release-it patch --disable-metrics",
     "update:deps": "ncu -u -x 'node-fetch' && pnpm install",
     "start:server": "serve node_modules/@metamask/test-dapp/dist -p 3000",
-    "start:contract": "cd contract && pnpm start",
-    "start:ui": "cd ui && pnpm dev",
+    "start:contract": "cd tools/contract && pnpm start",
+    "start:ui": "cd tools/ui && pnpm dev",
     "start:json-server": "json-server json-server-db.json --port 3004",
```

Comment:
should we use `dapp` as the folder name instead of `tools`? 
`dapp` is more intention revealing what the folder contents would be.